app/main.py

```diff
@@ -1,3 +1,64 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, float, '"Distance"']) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (float, int)):
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Union[int, float, '"Distance"']) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (float, int)):
+            self.km += other
+
+        return self
+
+    def __mul__(self, other: Union[int, float]) -> "Distance":
+        if isinstance(other, (float, int)):
+            return Distance(self.km * other)
+
+    def __truediv__(self, other: Union[int, float]) -> "Distance":
+        if isinstance(other, (float, int)):
+            return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Union[int, float, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        elif isinstance(other, (float, int)):
+            return self.km < other
+
+    def __gt__(self, other: Union[int, float, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        elif isinstance(other, (float, int)):
+            return self.km > other
+
+    def __eq__(self, other: Union[int, float, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        elif isinstance(other, (float, int)):
+            return self.km == other
+
+    def __le__(self, other: Union[int, float, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km <= other.km
+        elif isinstance(other, (float, int)):
+            return self.km <= other
+
+    def __ge__(self, other: Union[int, float, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km >= other.km
+        elif isinstance(other, (float, int)):
+            return self.km >= other
```

Comment:
try to rewrite these two methods in one line each. Use previously created magic methods
Hint: <= is the same as not >
Also remember to use operands directly: for example `<` instead of calling methods like this `self.__lt__(other)`